modules/top-level/output.nix

```diff
@@ -284,20 +302,24 @@ in
         '';
       };
 
+      # Set `wrapRc` and `impureRtp`s option defaults with even lower priority than `mkOptionDefault`
+      wrapRc = lib.mkOverride 1501 true;
+      impureRtp = lib.mkOverride 1501 false;
+
       extraConfigLuaPre = lib.mkOrder 100 (
-        # Add a global table at start of init
-        ''
-          -- Nixvim's internal module table
-          -- Can be used to share code throughout init.lua
-          local _M = {}
-        ''
-        + lib.optionalString config.wrapRc ''
-
-          -- Ignore the user lua configuration
-          vim.opt.runtimepath:remove(vim.fn.stdpath('config'))              -- ~/.config/nvim
-          vim.opt.runtimepath:remove(vim.fn.stdpath('config') .. "/after")  -- ~/.config/nvim/after
-          vim.opt.runtimepath:remove(vim.fn.stdpath('data') .. "/site")     -- ~/.local/share/nvim/site
-        ''
+        lib.concatStringsSep "\n" (
```

Comment:
Is the idea now that we want to remove the runtimepath stuff before the global table so that it isn't affected by anything from runtimepath?

Reply:
Moving the rtp _should_ be purely cosmetic. It just felt right to me that it be the very first thing in init.lua 🙃